tests/original_wb.prm

```diff
@@ -0,0 +1,6 @@
+# test that output/original.wb is written correctly
+
+# based on world_builder_simple.prm:
+# set World builder file = $ASPECT_SOURCE_DIR/tests/world_builder_simple.wb
+include $ASPECT_SOURCE_DIR/tests/world_builder_simple.prm
+set World builder file                     = $ASPECT_SOURCE_DIR/tests/world_builder_simple.wb
```

Comment:
you are missing a newline at the end of the file

Reply:
I always forget that this triggers the indentation to fail! Thank you for saving me the headache